notebooks/text-to-image-genai/text-to-image-genai.ipynb

```diff
@@ -0,0 +1,393 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    %pip install -q Pillow "diffusers==0.30.3" "gradio&gt;=4.19"
</pre><p>why you hardcode diffusers version?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/2472/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
removed